### ISSUE TITLE
[PYTHON] Path format for parameter does not work / Connexion Upgrade

### ISSUE THREAD

**Issue by kcacciatore** [swagger-api/swagger-codegen#5626]
##### Description

swagger-codegen (2.2.3) uses connexion library version 1.0.129. This version of connexion does not process the format: path attribute of the parameters portion of an API declaration. So if defining a path such as:

```yaml
/service/{serviceId}/{ext}:
    post:
      tags:
      - "service"
      summary: "Process an extension request"
      description: "Hook for extending the API"
      operationId: "invoke_extension"
      produces:
      - "application/json"
      parameters:
      - name: serviceId
        in: path
        required: true
        type: string
      - name: ext
        in: path
        required: true
        type: string
        format: path
```
POSTing to something like /services/example/test will yield serviceid='example' and ext='test', but POSTing something like /services/example/test/more will yield a 404 error when the expected parameter values passed to the operation would be serviceid='example' and ext='test/more'

Upgrading to connexion 1.1.9 addresses this issue.

##### Swagger-codegen version

2.2.3

##### Swagger declaration file content or url
[gist](https://gist.github.com/kcacciatore/3491bc07706a59f5a0e0a2ead465ca8a)

##### Command line used for generation

generated from editor.swagger.io.

##### Steps to reproduce

1. generate server stub (python-flask) from editor.swagger.io
2. unzip
3. cd into directory
4. create virtual environment virtualenv  -p /opt/local/bin/python3 venv
5. set virtual env - source venv/bin/activate
6. install dependencies - pip install -r requirements.txt
7. run server - python3 -m swagger_server
8. post to http://localhost:8080/v1/service/example/ext - should return 200
9. post to http://localhost:8080/service/example/ext/test - should return 404

The last step should process successfully, and adding debug or changing the default response string to print the variable values should print  'example' as the serviceid and 'ext/test' as the extension.

##### Related issues

None that I've found

##### Suggest a Fix

Upgrading the connexion library - I've done this already and can submit a PR. There is a change in the encoder.py generation as well, since connexion is isolating provider specific components into separate modules, so the JSONEncoder has moved.

**Comment by wing328** [comment]
@kcacciatore I agree with you to upgrade connexion to the latest stable version 1.1.9: https://pypi.python.org/pypi/connexion

I wonder if you can file a PR so that we can review the fix.

**Comment by wing328** [comment]
Closed via #5645
